src/patternfly/components/Select/select.scss

```diff
@@ -57,9 +57,16 @@
   --pf-c-select__toggle-badge--PaddingLeft: var(--pf-global--spacer--md);
 
   // Toggle Arrow
-  --pf-c-select__toggle-arrow--MarginLeft: var(--pf-global--spacer--md);
+  --pf-c-select__toggle-arrow--MarginLeft: var(--pf-global--spacer--sm);
+  --pf-c-select__toggle-arrow--MarginRight: var(--pf-global--spacer--sm);
   --pf-c-select__toggle-arrow--with-clear--MarginLeft: var(--pf-global--spacer--xs);
 
+  // Toggle Button
+  --pf-c-select__toggle-button--PaddingRight: var(--pf-global--spacer--sm);
+  --pf-c-select__toggle-button--PaddingLeft: var(--pf-global--spacer--sm);
+  --pf-c-select__toggle-button--MarginLeft: #{pf-size-prem(-8px)};
```

Comment:
I don't think you always want a negative margin here? Looks like you only want it when it's next to the clear button? Otherwise I don't understand why you would want it when it's next to the `input[type="text"]`.

But I don't know that you want to do it that way anyways, it causes a weird overlap of the clear button and toggle arrow.

<img width="189" alt="screen shot 2019-03-04 at 3 48 15 pm" src="https://user-images.githubusercontent.com/35148959/53765260-239c3600-3e95-11e9-8ee8-7dc9e3f26d86.png">
<img width="171" alt="screen shot 2019-03-04 at 3 50 10 pm" src="https://user-images.githubusercontent.com/35148959/53765290-34e54280-3e95-11e9-86ac-36d6902124f6.png">

I wonder if it's worth asking design if there can be more space between those elements so they don't have to overlap like that. If not, I would just use a smaller padding on the sides the clear and arrow button meet, like how we do in the options menu. 

<img width="198" alt="screen shot 2019-03-04 at 3 50 44 pm" src="https://user-images.githubusercontent.com/35148959/53765391-7675ed80-3e95-11e9-8585-801abd476973.png">